### ISSUE TITLE
Please add a description in English

### ISSUE THREAD

**Issue by pcav** [neogeo-technologies/geofoncier-editeur-rfu-qgis#13]
So users will know what is this about.
Merci bien.

**Comment by m431m** [comment]
Done fe11585

**Comment by pcav** [comment]
Thanks, Please upload a new version.

**Comment by m431m** [comment]
Done :-) -> experimental version 0.2

**Comment by miquigti** [comment]
**Hello !!!**

I'm using QGIS 2.14 but the soft ware give a error minidump.
Searching the solution in internet, the people say try disconnect all connectors. I do. But when arrive at your connector give error minidump and I can't disconnect or desinstall.

How I can delete it, please?

Thanks,

**Comment by m431m** [comment]
Yes, this is a mistake I've recently fixed: 12a6a37.
In `<your user's home directory>/.qgis2/python/plugins/GeofoncierEditeurRFU/` 
open `editor_rfu_geofoncier.py`,
then comment (or delete) lines 67 and 68:

```
#        if self.iface:
#            self.iface.removeDockWidget(self.rfu)
```

**Comment by lucashrosa** [comment]
Hi m431m,
do you know if is there any official QGIS solution about minidump error?

I understood that commented lines just bypass the error, not solve them.
I asking because I found the same problem with many others dockwidgets that I really need remove in my projects.
